arch/arm/boot/dts/nxp/imx/imx6ul-ts7250v3.dtsi

```diff
@@ -46,15 +45,6 @@
 		reg = <0x80000000 0x20000000>;
 	};
 
-	// pc104 pins B15 and B17.
-	i2c5_gpio: i2c-104 {
-		compatible = "i2c-gpio";
-		scl-gpios = <&fpga_bank1 13 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
-		sda-gpios = <&fpga_bank1 8 (GPIO_ACTIVE_HIGH | GPIO_OPEN_DRAIN)>;
-		#address-cells = <1>;
-		#size-cells = <0>;
-	};
-
```

Comment:
Changes to this file are basically reverting the previous commit, 18e58a0998d76abf8e26dce446128e543851366c. If the 18e58a0998d76abf8e26dce446128e543851366c commit is not necessary, it should be squashed so that the lines are not added then removed in two back to back commits